pkg/terminal/command.go

```diff
@@ -1075,6 +1087,7 @@ func continueUntilCompleteNext(t *Term, state *api.DebuggerState, op string, sho
 		if shouldPrintFile {
 			printfile(t, state.CurrentThread.File, state.CurrentThread.Line, true)
 		}
+		t.onStop()
```

Comment:
For these cases with multiple returns we should just `defer t.onStop()`.

Reply:
Good idea, done.